R/generate_event_cc_addhypo.R

```diff
@@ -0,0 +1,64 @@
+#' generate event table in common control
+#'
+#' @param input_data input data in common control situation
+#' @param hypothesis comparison group
+#'
+#' @return
+#' @export
+#'
+#' @examples
+#' input_data <- data.frame(
+#'   Population = c("Experimental 1", "Experimental 2", "Experimental 3", "Control"),
+#'   IA = c(70, 75, 80, 85),
+#'   FA = c(135, 150, 165, 170)
+#' )
+#'
+#' hypothesis <- list(
+#'   H1 = "Experimental 1 vs. Control",
+#'   H2 = "Experimental 2 vs. Control",
+#'   H3 = "Experimental 1 vs. Experimental 2"
+#' )
+#'
+#' result_table <- generate_event_table_cc(input_data, hypothesis)
+#' sorted_data <- result_table[order(result_table$analysis), ]
```

Comment:
It looks like the output of `generate_event_table_cc` is sorted. Could you please check if there is a specific reason why the `sorted_data` is required in the example?